lib/connection/statement.js

```diff
@@ -553,6 +553,10 @@ function BaseStatement(
     return context.statementId;
   };
 
+  this.getQueryId = function ()
+  {
+    return context.statementId;
```

Comment:
do we really need the new functions in result and statement? are we planning to make one of them deprecated?

Reply:
At the first commit, I just replaced getStatementId to getQueryId. However, Harry worried about the existing users who use getStatementId. Therefore, we consider making a new function(getQueryId) which returns the same result is better. Please let me know if you want us to replace getStatementId to getQueryId

Reply:
we can add a new method but we should mark the old one as deprecated probably? we can also replace it with BCR if we want

but my question is: cannot we just live with statementId in the whole code? do we have to have the method named with queryId?

Reply:
ok, let's then add new method and mark the old one as deprecated (in nodejs you can do this by a comment probably) and until we delete it with the BCR release in the future test both methods since both can be used by our users